data/assets/scene/solarsystem/missions/jwst/jwst.asset

```diff
@@ -0,0 +1,315 @@
+        Path = "/Solar System/Missions/JWST",
+        Hidden = true,
+    }
+}
+
+local JWSTModel = {
+    Identifier = "JWSTModel",
+    Parent = JWSTRotation.Identifier,
+    TimeFrame = {
+        Type = "TimeFrameInterval",
+        Start = "2020 JAN 01",
+        End = "2024 JAN 01"
+    },
+    Transform = {
+        Rotation = {
+            Type = "StaticRotation",
+            Rotation = {
+                0, 0, 0
+            }
+        }
+    },
+    Renderable = {
+        Type = "RenderableModel",
+        GeometryFile = model .. "/JWSTFBX.osmodel",
+        ModelScale = "Foot",
+        InvertModelScale = true,
+        EnableAnimation = true,
+        AnimationStartTime = "2018 10 01 15:00:00", --TODO: Update theese when the new animation is finished
+        AnimationMode = "Once",
+        LightSources = {
+            {
+                Type = "SceneGraphLightSource",
+                Identifier = "Sun",
+                Node = sunTransforms.SolarSystemBarycenter.Identifier,
+                Intensity = 1.0
+            }
+        },
+        PerformShading = true,
+        DisableFaceCulling = true
+    },
+    GUI = {
+        Name = "James Webb Space Telescope",
+        Path = "/Solar System/Missions/JWST",
+    }
+}
+
+local JWSTTrail = {
+    Identifier = "JWSTTrail",
+    Parent = transforms.L2.Identifier,
+    Renderable = {
+        Type = "RenderableTrailOrbit",
+        Translation = {
+            Type = "SpiceTranslation",
+            Target = "JWST", 
+            Observer = "392", -- L2
+            Kernels = JWSTKernel .. "/jwst_horizons_20200101_20240101_v01.bsp"
+        },
+        Color = { 0.9, 0.9, 0.0 },
+        Period = 91.3105,
+        Resolution = 100
+    },
+    GUI = {
+        Name = "JWST Trail",
+        Path = "/Solar System/Missions/JWST"
+    }
+}
+
+local JWSTSunTrail = {
+    Identifier = "JWSTSunTrail",
+    Parent = sunTransforms.SolarSystemBarycenter.Identifier,
+    Renderable = {
+        Enabled = false,
+        Type = "RenderableTrailOrbit",
+        Translation = {
+            Type = "SpiceTranslation",
+            Target = "JWST",
+            Observer = "SUN",
+            Kernels = JWSTKernel .. "/jwst_horizons_20200101_20240101_v01.bsp"
+        },
+        Color = { 0.0, 0.9, 0.9 },
+        Period = 365.242,
+        Resolution = 1000
+    },
+    GUI = {
+        Name = "JWST Sun Trail",
+        Path = "/Solar System/Missions/JWST"
+    }
+}
+
+local JWSTFov = {
+    Identifier = "JWSTFov",
+    Parent = JWSTModel.Identifier,
+    Renderable = {
+        Enabled = false,
+        Type = "RenderablePrism",
+        Segments = 6,
+        Lines = 3,
+        Radius = 3.25,
+        LineWidth = 1.0,
+        Color = {
+            1.0, 1.0, 1.0
+        },
+        Length = 9.2E15
+    },
+    Transform = {
+        Rotation = {
+            Type = "StaticRotation",
+            Rotation = {
+                0, 0, 0.523598776 -- 30 deg around Z-axis in radians
```

Comment:
Would suggest:
`0, 0, 30 * 3.1415926 / 180.0` 
Then there is no need for the comment and we can later introduce.

I think this should wokr as well:
`0, 0, math.rad(30)`